model/src/main/java/org/mskcc/cmo/metadb/model/SampleMetadata.java

```diff
@@ -18,8 +18,10 @@
 import org.mskcc.cmo.metadb.model.igo.QcReport;
 import org.neo4j.ogm.annotation.GeneratedValue;
 import org.neo4j.ogm.annotation.Id;
+import org.neo4j.ogm.annotation.NodeEntity;
 import org.neo4j.ogm.annotation.typeconversion.Convert;
 
+@NodeEntity(label = "SampleMetadata")
```

Comment:
I think just `@NodeEntity` by itself should work since we don't need to apply a different label name like we do for `MetadbSample`, which we just label `Sample`

Reply:
👍